### PR TITLE
new style fixing

### LINKED ISSUE #178: new style fixing

### PR DESCRIPTION
**Pull request by 2016DigiTech**
Fixing new styling 

Fix #178 

Test URLs:https://new-style-fixing--tlcgrp--hlxsites.hlx.page/about-us
- Before: https://main--tlcgrp--hlxsites.hlx.page/
- After: https://new-style-fixing--tlcgrp--hlxsites.hlx.page/about-us

### CODE REVIEW

**Comment by 2016DigiTech** [comment]
@asthabh23 Please review #178

**Comment by asthabh23** [comment]
@2016DigiTech , can you confirm on the URLs please?
I think I updated it right, the before and after URLs need to contain the link to the page for which changes are added.

**Comment by 2016DigiTech** [comment]
@asthabh23 Please Review

**Comment by 2016DigiTech** [comment]
> @2016DigiTech , can you confirm on the URLs please? I think I updated it right, the before and after URLs need to contain the link to the page for which changes are added.

https://new-style-fixing--tlcgrp--hlxsites.hlx.page/about-us

File: styles/styles.css [hunk 1]
```diff
@@ -178,6 +183,30 @@ main .member-subscriptions h5 > a {
...
+  font-size: 24px;
+  color: #fff;
+  font-weight: 300;
+  border-radius: 24px;
+  display: table;
+  margin: 0 auto;
+  margin-top: 50px;
+  margin-bottom: 0;
+  padding: 11px 49px 12px 50px;
+  background-color: #00582e;
```
Comment (asthabh23) [inline styles/styles.css:197]: you could consider using variable names

File: styles/styles.css [hunk 2]
```diff
@@ -178,6 +183,30 @@ main .member-subscriptions h5 > a {
   text-decoration: underline;
 }
 
+main .history-vision-section h5 {
+  text-align: center;
+  font-size: 24px;
+  color: #fff;
```
Comment (asthabh23) [inline styles/styles.css:189]: you could consider using variable names

**Review by asthabh23 — APPROVED** [review]

**Pull request merged**